packages/components/tests/dummy/app/templates/components/badge.hbs

```diff
@@ -176,123 +176,4 @@
-      <ol>
-        <li class="default">filled</li>
-        <li>inverted</li>
-        <li>outlined</li>
-      </ol>
-    </dd>
-    <dt>color <code>enum</code></dt>
-    <dd>
-      <p>Acceptable values:</p>
-      <ol>
-        <li class="default">neutral</li>
-        <li>neutral-dark-mode</li>
-      </ol>
-    </dd>
-    <dt>text <code>string</code></dt>
-    <dd>
-      <p>The text value that should go in the badge counter.</p>
-    </dd>
-    <dt>...attributes</dt>
-    <dd>
-      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
-    </dd>
-  </dl>
-</section>
-
-<section>
-  <h3 class="dummy-h3" id="bc-how-to-use"><a href="#bc-how-to-use" class="dummy-link-section">§</a> How to use</h3>
-  <p class="dummy-paragraph">Invocation of the component would look something like this:</p>
-  {{! prettier-ignore-start }}
-  <CodeBlock
-    @language="markup"
-    @code='
-      <Hds::BadgeCount @text="3" @size="large" @type="inverted" />
-    '
-  />
-  {{! prettier-ignore-end }}
-</section>
-<section>
-  <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
-  <p class="dummy-paragraph">This component has been designed and implemented with accessibility in mind. When used as
-    recommended, there should not be any accessibility issues with this component.
-  </p>
-</section>
-
-<section data-test-percy>
-  <h3 class="dummy-h3" id="bc-showcase"><a href="#bc-showcase" class="dummy-link-section">§</a> Showcase</h3>
-
-  <h4 class="dummy-h4">Content</h4>
-  <div class="dummy-badge-base-sample">
-    <Hds::BadgeCount @text="3" />
-    <Hds::BadgeCount @text="99+" />
-    <Hds::BadgeCount @text="v1.2.3" />
-  </div>
-  <div class="dummy-badge-base-sample">
-    <p>This is a paragraph: <Hds::BadgeCount @text="3" /></p>
-  </div>
-
-  <h4 class="dummy-h4">Size</h4>
-  <div class="dummy-badge-base-sample">
-    {{#each @model.BADGE_COUNT_SIZES as |size|}}
-      <Hds::BadgeCount @text={{capitalize size}} @size={{size}} />
-    {{/each}}
-  </div>
-
-  <h4 class="dummy-h4">Type</h4>
-  <div class="dummy-badge-base-sample">
-    {{#each @model.BADGE_COUNT_TYPES as |type|}}
-      <Hds::BadgeCount @text={{capitalize type}} @type={{type}} />
-    {{/each}}
-  </div>
-
-  <h4 class="dummy-h4">Color:</h4>
-  {{#each @model.BADGE_COUNT_COLORS as |color|}}
-    <p class="dummy-h6">{{capitalize color}}</p>
-    <div class="dummy-badge-color-grid">
-      {{#each @model.BADGE_COUNT_SIZES as |size|}}
-        {{#each @model.BADGE_COUNT_TYPES as |type|}}
-          <div
-            class="dummy-badge-base-sample dummy-badge-base-sample--type-{{type}}
-              dummy-badge-base-sample--color-{{color}}"
-          >
-            <Hds::BadgeCount @text="3" @size={{size}} @type={{type}} @color={{color}} />
-          </div>
-        {{/each}}
-      {{/each}}
-    </div>
-  {{/each}}
-</section>
\ No newline at end of file
+{{! ###################### }}
```

Comment:
Notice: at the top of this page there is a link that needs to be updated now, and point to the `badge-count` route.

Instead of 
```
<p class="dummy-paragraph"><em>Notice: for collection enumeration or version number, use
      <a href="#bc-overview">BadgeCount</a>.</em></p>
```
needs to be
```
<p class="dummy-paragraph"><em>Notice: for collection enumeration or version number, use
      <LinkTo @route="components.badge-count">BadgeCount</LinkTo>.</em></p>
```

Reply:
Probably, I would add also a similar cross-link from `BadgeCount` to `Badge`.